src/main/kotlin/org/wa/rceditor/application/view/fragments/RelationItemFragment.kt

```diff
@@ -1,26 +1,25 @@
 package org.wa.rceditor.application.view.fragments
 
+import javafx.beans.property.StringProperty
 import javafx.scene.layout.Priority
 import org.wa.rceditor.application.Styles
-import org.wa.rceditor.application.model.RelationItem
-import org.wa.rceditor.application.model.RelationItemModel
 import org.wa.rceditor.application.viewmodel.MainViewModel
 import tornadofx.*
 
-class RelationItemFragment: ListCellFragment<RelationItem>() {
+class RelationItemFragment: ListCellFragment<StringProperty>() {
```

Comment:
I think you will be able to combine this list cell fragment with the one for contributors

Reply:
It will be possible when I'll find the way to get the items of the list inside this class.